news/2018-10-13-osu-world-cup-2018-registrations-open.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "osu! World Cup 2018: Registrations now open!"
+date: 2018-10-13 13:00:00 +0000
+---
+
+Circle it on your calendars, because OWC 2018 registrations are now open!
+
+![](/wiki/shared/news/banners/OWC_2018_banner.png)
+
+For more details, refer to the **[osu! World Cup 2018 wiki article](/wiki/Tournaments/OWC/2018/)**.
+You can discuss the event and follow its most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/739799)**.
+ 
+## Tournament Schedule
+
+| ------------------ | ----------------------- |
+| Registration       | Oct 13th - Oct 27th     |
```

Comment:
if is before the 27, it should end on 26?, this to correlate with the wiki table